osu.Game/Input/Bindings/DatabasedKeyBinding.cs

```diff
@@ -29,5 +29,8 @@ public int IntAction
             get => (int)Action;
             set => Action = value;
         }
+
+        [Column("SAction")]
```

Comment:
that's not a great column name; `ActionName` maybe better? (goes for the field too I suppose).

Reply:
Sounds good. Will be changed.